Extension/src/LanguageServer/cppBuildTaskProvider.ts

```diff
@@ -185,9 +185,9 @@ export class CppBuildTaskProvider implements TaskProvider {
 
         const scope: TaskScope = TaskScope.Workspace;
         const task: CppBuildTask = new Task(definition, scope, definition.label, CppBuildTaskProvider.CppBuildSourceStr,
-            new CustomExecution(async (): Promise<Pseudoterminal> =>
+            new CustomExecution(async (resolvedDefinition: TaskDefinition): Promise<Pseudoterminal> =>
```

Comment:
@sean-mcmanus I couldn't get this to work. 
Argument of type '(resolvedDefinition: TaskDefinition) => Promise<Pseudoterminal>' is not assignable to parameter of type '() => Thenable<Pseudoterminal>'.

Reply:
can you add a comment similar to this:
`// VS Code will pass task definition back with all variables resolved.`

Reply:
Ok

Reply:
Actually, I'm hesitant to add that comment, because it's described already in the comment for CustomExecution:

![image](https://user-images.githubusercontent.com/19859882/99727597-152cee00-2a6d-11eb-9253-71a0f8bbb7e5.png)

and the name of the resolvedDefinition implies variables are resolved.